code/backtracking/src/sudoku_solve/sudokusolve.cpp

```diff
@@ -90,6 +142,7 @@ int main()
      {0, 0, 0, 4, 1, 9, 0, 0, 5},
      {0, 0, 0, 0, 8, 0, 0, 7, 9}};
     printMat(mat);
+    initlookupvalues(mat);
```

Comment:
Please rename it with camel-case.

Reply:
Hey,  How can i edit that? .. I am new to Github, please help 

> Please rename it with camel-case.

Reply:
Just commit a new change and then push it to the same branch.

Reply:
Hey, 
Unrelated to this commit, I wanted to apply for Algorithms and data structures internship at opengenus, Can you help me atleast get to the screening  process?